### ISSUE TITLE
AppImage zbar qr code reader issues on Tails 3.12.1

### ISSUE THREAD

**Issue by JazzTp** [spesmilo/electrum#5129]
_Please_ let me know if reporting issues related with Tails is not useful, it's quite possible that the developer @mithrandi (about whom I read in #5083) already regularly tests Electrum versions on Tails versions, so I would be unusefully generating noise and work for other developers here.

## electrum-3.3.4-x86_64.AppImage
- Electrum starts fine.
- The QRcode scanner window opens but the QRcode scanner doesn't start (it did on Tails 3.12, after a long while, 30-120 seconds maybe, I wasn't watching the screen any more and noticed a bit later, and it was displaying a new frame once in a while, say each 3-5 seconds).
- Closing the QRcode scanner window => segmentation fault (both on Tails 3.12 and 3.12.1)

(Works fine in Ubuntu 18.04.)

## Electrum-3.3.4.tar.gz
- Syntax error, already reported in #5008, issue which was closed as due to the Python version coming with Tails, too old.

```
File "./run_electrum", line 53
    sys.exit(f"Error: {str(e)}. Try 'sudo python3 -m pip install <module-name>'")
                                                                               ^
SyntaxError: invalid syntax
```

(Works fine in Ubuntu 18.04.)

**Comment by SomberNight** [comment]
The Python version issue is not that interesting, it's due to Tails/Debian having too old Python. Still, I've tried to make the error text more readable in  https://github.com/spesmilo/electrum/commit/0de954546ac82fc4b2c474567493b5afb9698a19

Re the AppImage qr code reader issues, thank you for reporting it! I'll try to have a look later.

**Comment by JazzTp** [comment]
@SomberNight **Connecting the webcam into another USB port "solved" the porblem**, the QRcode scanner in **electrum-3.3.4-x86_64.AppImage** is now opening fine and the image refresh is fluid.

This USB port is  on the back of the PC as the one I was using before, I haven't the slightest idea what might be different between the two ports, I wasn't and I'm not connecting the webcam through a hub. (Actually into the other port I have now plugged the pendrive where I keep the Electrum wallets on which I'm testing and it's working fine.)

Feel free to close this issue if you evaluate that no further invastigation is useful / feasible (sorry if I wasted your time).
